src/compiler/transformers/static-to-meta/component.ts

```diff
@@ -18,14 +18,28 @@ import { parseStringLiteral } from './string-literal';
 import { setComponentBuildConditionals } from '../component-build-conditionals';
 import ts from 'typescript';
 
+/**
+ * Given an instance of typescript's IR for a class decoration ({@see ts.ClassDeclaration})
+ * which represents a Stencil component class declaration, parse and format
+ * various pieces of data about static class members which we use in the
+ * compilation process
+ *
+ * @param compilerCtx the current compiler context
+ * @param typeChecker a typescript typechecker instance
```

Comment:
```suggestion
 * @param typeChecker a typescript type checker instance
```

Reply:
`"type-checker"`? another possibility haha

`"type checker"` sounds good to me